### PR TITLE
add revcomp_i5 task and bump reports docker

### PR DESCRIPTION
**Pull request by dpark01**

### CODE REVIEW

File: pipes/WDL/tasks/tasks_demux.wdl [hunk 1]
```diff
@@ -79,6 +79,48 @@ task samplesheet_rename_ids {
...
+    with open(old_sheet, "rt") as inf:
+      with open(new_base+'.tsv', 'wt') as outf:
+        if revcomp:
+          sheet = csv.DictReader(inf, delimiter='\t')
+          writer = csv.DictWriter(outf, sheet.fieldnames, delimiter='\t', dialect=csv.unix_dialect, quoting=csv.QUOTE_MINIMAL)
+          writer.writeheader()
+          for row in sheet:
+            if row.get('barcode_2')
+              and all(s in 'ATGC' for s in row['barcode_2']):
+              row['barcode_2'] = str(Bio.Seq.Seq(row['barcode_2']).reverse_complement())
```
Comment (tomkinsc) [inline pipes/WDL/tasks/tasks_demux.wdl:105]: Not that biopython increases the size of the docker image much, but builtins could be used for reverse complementing via `bytearray.maketrans()` (ex. [here](https://github.com/broadinstitute/viral-core/blob/612d269e75905e7b747f67ef56ba120d76f2306b/util/illumina_indices.py#L1524-L1529); could extend to ambiguity codes.). Probably faster too, though that's not a concern for sample sheets.

**Comment by tomkinsc** [comment]
This will come in handy! Especially if we automatically re-try demux with a rev-comped sample sheet for failing runs where that's a plausible fix.

**Comment by dpark01** [comment]
Yeah then we just have to detect the failure mode reliably and then we can use this to try it…

File: pipes/WDL/tasks/tasks_demux.wdl (hunk 1 above)
Comment (dpark01) [reply pipes/WDL/tasks/tasks_demux.wdl:105]: well the general purpose py3-bio docker image already had it, so I figured I'd keep it simple

**Pull request merged**